cg/meta/report/templates/balsamic_report.html

```diff
@@ -168,15 +168,17 @@ <h4>Kundinformation</h4>
                 <th>Läspar [M]</th>
                 <th>Mediantäckning [baser]</th>
               {% if "helgenomsekvensering" in case.data_analysis.type %}
-                  {% if "normal" in case.data_analysis.type %}
-                    <th>Täckningsgrad 15x [%]</th>
-                    <th>Täckningsgrad 60x [%]</th>
-                  {% else %}
-                    <th>Täckningsgrad 60x [%]</th>
-                  {% endif %}
+                {% if "normal" in case.data_analysis.type %}
+                  <th>Täckningsgrad 15x [%]</th>
+                  <th>Täckningsgrad 60x [%]</th>
+                {% else %}
+                  <th>Täckningsgrad 60x [%]</th>
+                {% endif %}
+                <th>PCT_PF_READS_IMPROPER_PAIRS [%]</th>
```

Comment:
Do we want to change this one to the Swedish one as well?

Reply:
Yes! Thanks!